agent/agent.go

```diff
@@ -159,8 +163,14 @@ func (a *Agent) index(input interface{}, ctx common.RestContext) (interface{}, e
 // 2. Ensures interhost routes are in place
 // 3. Creates ip route pointing new interface
 // 4. Provisions firewall rules
-func (a *Agent) k8sPodUpHandle(netif NetIf) error {
-	log.Print("Agent: processing request to provision new interface")
+func (a *Agent) k8sPodUpHandle(netReq NetworkRequest) error {
```

Comment:
k8sPodUpHandle() and k8sPodUpHandler(). Very similar names.

Could we rename k8sPodUpHandle() to k8sPodUpProvision() or k8sPodUpProcessing() ?